website/versioned_docs/version-27.0/CLI.md

```diff
@@ -262,6 +262,10 @@ Alias: `-o`. Attempts to identify which tests to run based on which files have c
 
 Allows the test suite to pass when no files are found.
 
+### `--platform`
+
+Specifies which platform to target. Overrides `haste.defaultPlatform` option.
+
```

Comment:
I believe this should also go into `docs/CLI.md` so that it's included in future versions of the docs as well